evaluate/task/repository.go

```diff
@@ -50,27 +72,21 @@ type Repository struct {
 
 var _ task.Repository = (*Repository)(nil)
 
+// RepositoryConfigurationFileName holds the file name for a repository configuration.
+const RepositoryConfigurationFileName = "repository.json"
```

Comment:
I would do a check inside `LoadRepositoryConfiguration` to see if the path has the "file" at the end and append it if not, so it would be possible to either define a directory or a file for `LoadRepositoryConfiguration` that way we would have fewer unnecessary exposed variables and fewer `filepath.Join` stuff.